general-configuration-guide/upload-images.md

```diff
@@ -7,13 +7,11 @@ keywords: uploader uploadImages image handler
 
 {% include pardot-px-moxiemanager.html %}
 
-> Note: This image upload feature is available for TinyMCE version 4.3 and above. Alternatively, the [PowerPaste plugin]({{ site.baseurl }}/plugins/powerpaste/) is capable of this functionality in versions of TinyMCE 4.0 and above.
-
-The image uploader is designed to complement the new image editing functionality of TinyMCE. Images that are edited within TinyMCE can be uploaded using this function. Local images that are added through other means, for example drag and drop when using the [paste_data_images]({{ site.baseurl }}/plugins/paste/#paste_data_images) configuration property, or using Tiny's PowerPaste Plugin.
+The image uploader is designed to complement the image editing functionality of {{site.productname}}. Images that are edited within {{site.productname}} can be uploaded using this function. Local images that are added through other means are also uploaded using this function, such as images added by drag and drop when using the [paste_data_images]({{ site.baseurl }}/plugins/paste/#paste_data_images) configuration property, or using the {{site.companyname}} [PowerPaste plugin]({{ site.baseurl }}/plugins/powerpaste/).
 
 Once uploaded, TinyMCE automatically updates the `<image>` src attribute with the new path to the remote image.
 
-Local images can be uploaded to TinyMCE through the use of the new `editor.uploadImages()` function. This functionality is handled asynchronously, meaning that it is possible for users to save their content before all images have completed uploading.  If this occurs and no server path to the remote image is available, the images are saved as Base 64.
+Local images can be uploaded to {{site.productname}} through the use of the `editor.uploadImages()` function. This functionality is handled asynchronously, meaning that it is possible for users to save their content before all images have completed uploading.  If this occurs and no server path to the remote image is available, the images are saved as Base 64.
```

Comment:
small thing - it's usually "base64" with no space

Reply:
Yeah. I'll be revising this page next year